Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -1397,6 +1608,230 @@ script:
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: deletion_protection
+        description: 'Weather the DB instance has deletion protection enabled. The database can’t be deleted when deletion protection is enabled. By default, deletion protection isn’t enabled. For more information, see Deleting a DB Instance.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-enable-auto-upgrade-quick-action
+    description: Enables automatic minor version upgrades for the RDS database instance by running the aws rds modify-db-instance command. Successful execution requires the rds:ModifyDBInstance permission.
+    prettyname: Enable RDS Auto Upgrade
+    hidden: true
+    quickaction: true
+    arguments:
+      - name: account_id
+        description: The AWS account ID.
+        prettyname: Account ID
+        required: true
+        prettypredefined:
+          Account ID: ${asset.xdm.asset.realm}
+      - name: region
+        description: The AWS region.
+        prettyname: Region
+        required: true
+        prettypredefined:
+          Region: ${asset.xdm.asset.cloud.region}
+      - name: db_instance_identifier
+        prettyname: RDS Instance ID
+        description: 'The identifier of DB instance to modify.'
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: auto_minor_version_upgrade
+        description: 'Weather minor version upgrades are applied automatically to the DB instance during the maintenance window. An outage occurs when all the following conditions are met: The automatic upgrade is enabled for the maintenance window. A newer minor version is available. RDS has enabled automatic patching for the engine version. If any of the preceding conditions isn’t met, Amazon RDS applies the change as soon as possible and doesn’t cause an outage. For an RDS Custom DB instance, don’t enable this setting. Otherwise, the operation returns an error.'
+        defaultValue: 'true'
+        hidden: true
+      - name: apply_immediately
+        description: 'Weather the modifications in this request and any pending modifications are asynchronously applied as soon as possible, regardless of the PreferredMaintenanceWindow setting for the DB instance. By default, this parameter is disabled. If this parameter is disabled, changes to the DB instance are applied during the next maintenance window. Some parameter changes can cause an outage and are applied on the next call to RebootDBInstance , or the next failure reboot.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-enable-multi-az-quick-action
+    description: Enables Multi-AZ deployment for the RDS database instance to enhance availability and durability by running the aws rds modify-db-instance command. Successful execution requires the rds:ModifyDBInstance permission.
```

```suggestion
    description: Enables Multi-AZ deployment for the RDS database instance to enhance availability and durability by running the aws rds modify-db-instance command.
```
Replace aws rds modify-db-instance.